### ISSUE TITLE
How To Force The CPD ?

### ISSUE THREAD

**Issue by manar-mani** [openNDS/openNDS#453]
Is it possible to force the captive portal to be always opened on chrome and not the default one ?

**Comment by bluewavenet** [comment]
@manar-mani 
No. The CPD is controlled entirely by the operating system on the client device (iOS, Android etc).
See:
https://opennds.readthedocs.io/en/stable/howitworks.html#captive-portal-detection-cpd
